unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Catalog/SceneObjectDropController.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class SceneObjectDropController : MonoBehaviour
```

Comment:
This has to be a `MonoBehaviour`? `catalogGroupListView` can't be provided from elsewhere? Remember we are moving away from unneeded `MonoBehaviour` components. If this component isn't absolutely necessary we most likely will have to restructure it later.

Reply:
Yes it is needed because the function SceneObjectDropped must be called from an `event trigger` component that is associated with an image, of course, it can always be refactored

Reply:
Changed!